SSffmpegVideoOperation/src/main/java/com/simform/videooperations/CallBackOfQuery.kt

```diff
@@ -34,30 +36,39 @@ public class CallBackOfQuery {
         FFmpeg.cancel()
     }
 
-    private fun process(context: AppCompatActivity, query: Array<String>, ffmpegCallBack: FFmpegCallBack) {
+    private fun process(query: Array<String>, ffmpegCallBack: FFmpegCallBack) {
+        val processHandler = Handler(Looper.getMainLooper())
         Config.enableLogCallback { logMessage ->
             val logs = LogMessage(logMessage.executionId, logMessage.level, logMessage.text)
-            ffmpegCallBack.process(logs)
+            processHandler.post {
+                ffmpegCallBack.process(logs)
+            }
         }
         Config.enableStatisticsCallback { statistics ->
             val statisticsLog =
                 Statistics(statistics.executionId, statistics.videoFrameNumber, statistics.videoFps, statistics.videoQuality, statistics.size, statistics.time, statistics.bitrate, statistics.speed)
-            ffmpegCallBack.statisticsProcess(statisticsLog)
+            processHandler.post {
+                ffmpegCallBack.statisticsProcess(statisticsLog)
+            }
         }
         when (FFmpeg.execute(query)) {
             Config.RETURN_CODE_SUCCESS -> {
-                context.runOnUiThread {
+                processHandler.post {
                     ffmpegCallBack.success()
                 }
             }
             Config.RETURN_CODE_CANCEL -> {
-                ffmpegCallBack.cancel()
-                FFmpeg.cancel()
+                processHandler.post {
+                    ffmpegCallBack.cancel()
+                    FFmpeg.cancel()
+                }
             }
             else -> {
-                ffmpegCallBack.failed()
-                Config.printLastCommandOutput(Log.INFO)
-            }
+                processHandler.post {
+                    ffmpegCallBack.failed()
+                    Config.printLastCommandOutput(Log.INFO)
+                }
+                }
```

Comment:
Please do the code in format.

Reply:
@adfleshner : Also, please update library version

Reply:
@adfleshner: Also, I have merged 1 PR, so rebase your code with latest one